diff.go

```diff
@@ -16,7 +17,7 @@ func NewDiffString(diff string, strip int) DiffService {
 	return &DiffString{b: []byte(diff), strip: strip}
 }
 
-func (d *DiffString) Diff() ([]byte, error) {
+func (d *DiffString) Diff(_ context.Context) ([]byte, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method DiffString.Diff should have comment or be unexported